vendor/modules.txt

```diff
@@ -299,8 +299,11 @@ k8s.io/client-go/util/keyutil
 k8s.io/client-go/util/workqueue
 # k8s.io/klog/v2 v2.4.0
 k8s.io/klog/v2
-# k8s.io/utils v0.0.0-20201110183641-67b214c5f920
+# k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+## explicit
 k8s.io/utils/buffer
+k8s.io/utils/clock
```

Comment:
Well looks like you implemented your own fake clock so this dependency is no longer needed :)

Reply:
Removed.